src/components/containers/ContainersFilters.js

```diff
@@ -20,30 +20,36 @@ const ContainersFilters = ({
   setFilter,
   list,
 }) => {
+  const containerKindsItem = CONTAINER_FILTERS.kind__in
 
   const onChangeFilter = (filter, value) => {
     setFilter(filter.key, value)
     setTimeout(() => {list()}, 500)
   }
 
-  const containerBarcodeItem = CONTAINER_FILTERS.barcode__icontains
-  const containerKindsItem = CONTAINER_FILTERS.kind__in
+  // Container KINDS are retrieved from containerKinds state store.
+  // It had to be separated here in order to keep the constants.js file independent from the store
+  const containersFiltersWithoutKind = () => {
+    let containersFiltersCopy = Object.assign({}, CONTAINER_FILTERS);
+    delete containersFiltersCopy.kind__in;
+    return containersFiltersCopy;
+  }
```

Comment:
You're looking for [dissoc(key, object)](https://selfrefactor.github.io/rambda/#/?id=dissoc)